src/Components/Locations.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+
+export default class Locations extends Component {
+    constructor(){
+        super()
+        this.state = {
+            locations: [],
+            showList: false,
+        }
+    }
+    getLocations = async () => {
+        const data = await axios.get("https://ghibliapi.herokuapp.com/locations")
+        console.log(data.data)
+        this.setState({
+            locations: data.data.map((location) => location.name)
+        })
+    }
+    componentDidMount() {
+        this.getLocations();
+    }
+    handleClick = () => {
+        this.setState({
+            showList: (!this.state.showList),
+        })
+    }
+    render() {
+        const listItems = this.state.locations.map((location, i) => <li key={i}>{location}</li>) 
```

Comment:
If possible, it's always preferable to use a unique id (like location.id) as the key instead of an array index.